ZipBuilder/Sources/sem-versions/PackageReader/CocoapodsReader.swift

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+import ShellUtils
+
+import PathKit
+
+class CocoapodsReader: PackageReader {
+  func packagesInDirectory(_ dirURL: URL) throws -> [PackageData] {
+    return try podspecURLs(at: dirURL)
+      .compactMap { (podspecURL) -> PackageData? in
+        print("podspecURL: \(podspecURL)")
+        return try parsePodspec(at: podspecURL).packageData(baseDir: dirURL)
+      }
+  }
+
+  private func podspecURLs(at dirURL: URL) throws -> [URL] {
```

Comment:
Nit: this should probably match the above style, being `podspecsInDirectory(_ dirURL: URL)`